### DISCUSSION TITLE
Do you plan to release a new NuGet version?

### DISCUSSION THREAD

**Discussion by DrFrankinStein** [jdomnitz/net-mdns#21] [Q&A] [answered]
I like to use your changes related to IMulticastService interface you have added recently but they are not included in the last released NuGet 0.30.0.

And also thanks for your maintaining this project !

**Comment by jdomnitz** [comment, marked as answer]
Great catch.  Updated and released as v0.31.0

**Reply by DrFrankinStein** [reply]
Thanks you very much!
